kubectl-plugin/src/resources/tests.rs

```diff
@@ -0,0 +1,178 @@
+}
+
+#[tokio::test]
+async fn get_pools() {
+    if CLUSTER.get().is_none() {
+        setup().await;
+    }
+    let pools = CLUSTER
+        .get()
+        .unwrap()
+        .rest_v00()
+        .pools_api()
+        .get_pools()
+        .await
+        .unwrap();
+    compare(format!(" ID                 TOTAL CAPACITY  USED CAPACITY  DISKS                                                                  NODE        STATUS  MANAGED \n mayastor-1-pool-1  100663296       8388608        {}  mayastor-1  Online  true \n", pools[0].state.as_ref().unwrap().disks[0].clone()), pools);
+}
+
+#[tokio::test]
+async fn get_pool() {
+    if CLUSTER.get().is_none() {
+        setup().await;
+    }
+    let pool = CLUSTER
+        .get()
+        .unwrap()
+        .rest_v00()
+        .pools_api()
+        .get_pool(CLUSTER.get().unwrap().pool(0, 0).as_str())
+        .await
+        .unwrap();
+    compare(format!(" ID                 TOTAL CAPACITY  USED CAPACITY  DISKS                                                                  NODE        STATUS  MANAGED \n mayastor-1-pool-1  100663296       8388608        {}  mayastor-1  Online  true \n", pool.state.as_ref().unwrap().disks[0].clone()), pool);
+}
+
+#[tokio::test]
+async fn get_nodes() {
+    if CLUSTER.get().is_none() {
+        setup().await;
+    }
+    let nodes = CLUSTER
+        .get()
+        .unwrap()
+        .rest_v00()
+        .nodes_api()
+        .get_nodes()
+        .await
+        .unwrap();
+    compare(
+        format!(
+            " ID          GRPC ENDPOINT   STATUS \n mayastor-1  {}  Online \n",
+            nodes[0].state.as_ref().unwrap().grpc_endpoint
+        ),
+        nodes,
+    );
+}
+
+#[tokio::test]
+async fn get_node() {
+    if CLUSTER.get().is_none() {
+        setup().await;
+    }
+    let node = CLUSTER
+        .get()
+        .unwrap()
+        .rest_v00()
+        .nodes_api()
+        .get_node(CLUSTER.get().unwrap().node(0).as_str())
+        .await
+        .unwrap();
+    compare(
+        format!(
+            " ID          GRPC ENDPOINT   STATUS \n mayastor-1  {}  Online \n",
+            node.state.as_ref().unwrap().grpc_endpoint
+        ),
+        node,
+    );
+}
+
+#[tokio::test]
+async fn get_replica_topology() {
+    if CLUSTER.get().is_none() {
+        setup().await;
+    }
+    let replica_topo = CLUSTER
+        .get()
+        .unwrap()
+        .rest_v00()
+        .volumes_api()
+        .get_volume(&Uuid::parse_str(VOLUME_UUID).unwrap())
+        .await
+        .unwrap()
+        .state
+        .replica_topology;
+    let replica_ids: Vec<String> = replica_topo.clone().into_keys().collect();
+    compare(format!(" ID                                    NODE        POOL               STATUS \n {}  mayastor-1  mayastor-1-pool-1  Online \n", replica_ids[0]), replica_topo);
```

Comment:
I tried to break it as much as possible. The String with headers cannot be broken to different lines the tests get failed.

Reply:
Maybe it can be changed to use "fill/alignment" instead? - https://doc.rust-lang.org/std/fmt/#fillalignment

Reply:
could we not just do something like:
```rust
let header = "ID REPLICAS TARGET-NODE ACCESSIBILITY STATUS SIZE \n";
let row = format!("{} .....");
compare( header + row );
```rust

Reply:
> Maybe it can be changed to use "fill/alignment" instead? - https://doc.rust-lang.org/std/fmt/#fillalignment

I used this and created something, The space between the HEADERS is `2` spaces and if the value under that HEADER exceeds the HEADER length then the space is `len(value)-len(HEADER)+2` spaces. I followed this pattern and used the fill/alignment. PTAL if this seems good.